.github/workflows/js-css-lint.yml

```diff
@@ -5,7 +5,31 @@ on:
     branches:
       - develop
       - main
+    # Only run if CSS/JS/MD-related files changed.
+    paths:
+      - '.github/workflows/js-css-lint.yml'
+      - '**.js'
+      - '**.json'
```

Comment:
Similar to my comment on the JS tests workflow, we should exclude irrelevant json files such as `composer.json`. See other comment.